arches/app/permissions/arches_permission_base.py

```diff
@@ -389,7 +250,12 @@ def get_users_for_object(self, perm: str, obj: Model) -> list[User]:
 
         ret = []
         for user in User.objects.all():
-            if user.has_perm(perm, obj):
+            default_perms = self.get_default_permissions(user, obj)
+            if (
+                perm
+                in self.get_user_perms(user, obj).values_list("codename", flat=True)
```

Comment:
self.get_user_perms only takes resource instances, but `get_users_with_permission_for_object` is applied for nodegroups in the MapLayerManagerView. This causes an error when attempting to access the map layer manager.